packages/fast-colors/src/rgb.ts

```diff
@@ -0,0 +1,90 @@
+
+export function hexToRgb(hex: string): string {
+  let processedHex: IRgbJSON;
+  processedHex = properRGBValues(hex);
+  let r: string;
+  let g: string;
+  let b: string;
+  let a: string;
+  let rgb: boolean;
+  let rgba: boolean;
+
+  r = processedHex.r;
+  g = processedHex.g;
+  b = processedHex.b;
+  a = processedHex.a;
+  rgb = processedHex.rgb;
+  rgba = processedHex.rgba;
+
+  if (r.length === 1) {
+    r += r;
+  }
+  if (g.length === 1) {
+    g += g;
+  }
+  if (b.length === 1) {
+    b += b;
+  }
+
+  if (rgb) {
+    return `rgb(${parseInt(r, 16)}, ${parseInt(g, 16)}, ${parseInt(b, 16)})`;
+  } else if (rgba) {
+    if (a.length === 1) {
+      a += a;
+    }
+    return `rgba(${parseInt(r, 16)}, ${parseInt(g, 16)}, ${parseInt(b, 16)}, ${(parseInt(a, 16) / 255)})`;
+  }
+}
+
+interface IRgbJSON {
```

Comment:
We've also changed our naming convention for interfaces - would you mind removing the `I` from the interface name?

Reply:
Hi Nicholas. It seems that if I remove the `I` from the name, I get an error. I've synced with upstream. Should I still remove it?